<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addChVidWidg</class>
 <widget class="QWidget" name="addChVidWidg">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>565</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat Medium</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
     background: rgb(30, 30, 30);
	color: white;

}
</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton:hover{
	color: rgb(190,0,0);
}
</string>
   </property>
   <property name="text">
    <string>Товар доступен на складе</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFullName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Наименование видеокарты</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>841</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-top: 1px solid rgb(180, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>841</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-top: 1px solid rgb(180, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comBoxVidPost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	background-color: rgb(40,40,40);
	border: 1px;
	border-bottom: 1px solid red;
	padding-left: 5px;
	selection-background-color: rgb(105, 0, 0);
}

QComboBox QListView{
	border: 1px solid red;
	padding-left: 5px;
	selection-background-color: rgb(105, 0, 0);
}

QComboBox:hover{
	background-color: rgb(55,55,55);
}

QComboBox::drop-down{
border: 0px;
}

QComboBox::down-arrow {
border-image: url(&quot;E:/pcconf/images/down-arrow.png&quot;);
width: 17px;
height: 17px;
margin-right: 5px;
}

QComboBox::down-arrow:on {
	border-image: url(&quot;E:/pcconf/images/down-arrow (1).png&quot;);
	width: 17px;
	height: 17px;
	margin-right: 5px;
}

QComboBox:on{
border: 2px solid rgb(100,0,0);
}

QComboBox QAbstractItemView {
	background: rgb(250, 250, 250);
	border-width: 1px;
	border-style: solid;
	border-color: rgb(235, 235, 235);
	selection-background-color: rgb(240, 240, 240);
	selection-color: rgb(25, 25, 25);
}
</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Gigabyte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nvidia</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbVidPost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Поставщик</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Видеокарта</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVidSave">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-color: rgb(30,30,30);
	background-color: rgb(40,40,40);
	color: #fffafa;
}

QPushButton::hover
{
	background-color: rgb(60,60,60);
	border-color: rgb(40,40,40);
	border-bottom: 1px solid  green;
}
</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVidCancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-color: rgb(30,30,30);
	background-color: rgb(40,40,40);
	color: #fffafa;
}

QPushButton::hover
{
	background-color: rgb(60,60,60);
	border-color: rgb(40,40,40);
	border-bottom: 1px solid  red;
	color: red;
}
</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidVolume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Объём памяти [Мб]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Тип памяти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Частота памяти [МГц]</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidFreq">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Тип интерфейса</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Мониторы</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidResol">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Разрешение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Длина [см]</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidLength">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidTdp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Тепловыделение (TDP)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidKol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidPrice">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Цена [руб.]</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>841</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
border-top: 1px solid rgb(180, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comBoxVidInterface">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	background-color: rgb(40,40,40);
	border: 1px;
	border-bottom: 1px solid rgb(120,120,120);
	padding-left: 5px;
}

QComboBox:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid red;
}

QComboBox::drop-down{
border: 0px;
}

QComboBox::down-arrow {
border-image: url(&quot;E:/pcconf/images/down-arrow (2).png&quot;);
width: 17px;
height: 17px;
margin-right: 5px;
}

QComboBox::down-arrow:on {
	border-image: url(&quot;E:/pcconf/images/down-arrow (3).png&quot;);
	width: 17px;
	height: 17px;
	margin-right: 5px;
}

QComboBox:on{
border: 2px solid rgb(100,0,0);
}

QComboBox QAbstractItemView {
	background: rgb(250, 250, 250);
	border-width: 1px;
	border-style: solid;
	border-color: rgb(235, 235, 235);
	selection-background-color: rgb(240, 240, 240);
	selection-color: rgb(25, 25, 25);
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>PCI-E 2.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PCI-E 3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PCI-E 4.0</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comBoxVidMonitor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	background-color: rgb(40,40,40);
	border: 1px;
	border-bottom: 1px solid rgb(120,120,120);
	padding-left: 5px;
}

QComboBox:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid red;
}

QComboBox::drop-down{
border: 0px;
}

QComboBox::down-arrow {
border-image: url(&quot;E:/pcconf/images/down-arrow (2).png&quot;);
width: 17px;
height: 17px;
margin-right: 5px;
}

QComboBox::down-arrow:on {
	border-image: url(&quot;E:/pcconf/images/down-arrow (3).png&quot;);
	width: 17px;
	height: 17px;
	margin-right: 5px;
}

QComboBox:on{
border: 2px solid rgb(100,0,0);
}

QComboBox QAbstractItemView {
	background: rgb(250, 250, 250);
	border-width: 1px;
	border-style: solid;
	border-color: rgb(235, 235, 235);
	selection-background-color: rgb(240, 240, 240);
	selection-color: rgb(25, 25, 25);
}
</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="teVidType">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbChip">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string>Наименование чипа</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVidChip">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
	padding-left: 5px;
	border: 1px solid rgb(20,20,20);
	background-color: rgb(40,40,40);
	border-bottom: 1px solid  rgb(120,120,120);
	selection-background-color: rgb(105, 0, 0);
}

QTextEdit:hover{
	background-color: rgb(55,55,55);
	border-bottom: 1px solid  red;
}</string>
   </property>
  </widget>
  <zorder>radioButton</zorder>
  <zorder>lbFullName</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>comBoxVidPost</zorder>
  <zorder>lbVidPost</zorder>
  <zorder>teVidName</zorder>
  <zorder>label_3</zorder>
  <zorder>btnVidSave</zorder>
  <zorder>btnVidCancel</zorder>
  <zorder>teVidVolume</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>teVidFreq</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>teVidResol</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>teVidLength</zorder>
  <zorder>teVidTdp</zorder>
  <zorder>label_11</zorder>
  <zorder>teVidKol</zorder>
  <zorder>label_12</zorder>
  <zorder>teVidPrice</zorder>
  <zorder>label_13</zorder>
  <zorder>line_3</zorder>
  <zorder>label_5</zorder>
  <zorder>comBoxVidInterface</zorder>
  <zorder>comBoxVidMonitor</zorder>
  <zorder>teVidType</zorder>
  <zorder>lbChip</zorder>
  <zorder>teVidChip</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
